participants/dataMain.md

```diff
@@ -0,0 +1,69 @@
+# <Data main>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+**Our project name is Data main, We'are simon, Allen and Joe in Slack**
+
+## (1) Brief Project Description
+
+
+- Users can find and download mirror files and Data sets from Data Main.
+- Data Main aims to create an open community of mirrored files and Data sets.
+
+## (2) Link to Project UI
+
+- [ui](https://github.com/cuiwenbian/dataMain-info/tree/master/images)
+- [source](https://github.com/cuiwenbian/dataMain)
+
+
+## (3) What does your application/UI do?
+
+Users can download linux images or other AI data.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+**ISO**
```

Comment:
Can you open a PR to add this to the curated dataset?   What is your source for ISO images?

Reply:
this one #171